### DISCUSSION TITLE
Syntax for Lat & Long to Census Tract Look Up

### DISCUSSION THREAD

**Discussion by cymonegates** [jessecambon/tidygeo#165] [Q&A]
This will be my last question, so thank you for your help in advance. I'm trying to recreate the "Find Geographic Coordinates" search option using lat and long with the Census Geocoder to get the census tract information (link below). How would I do this in tidygeocoder syntax? I've tried a few methods using the Census API guidance but can't seem to get it working. The census documentation suggests some combination of x,y (searchtype = coordinates).

https://geocoding.geo.census.gov/geocoder/geographies/coordinates?x=-111.8681&y=33.39545&benchmark=4&vintage=410

**Comment by jessecambon** [comment]
The `reverse_geo()` and `reverse_geocode()` functions should be what you are looking for: https://jessecambon.github.io/tidygeocoder/reference/reverse_geo.html

**Reply by cymonegates** [reply]
I'm not seeing a way to use reverse_geo() or reverse_geocode() to get census tract number. ""census" is not a valid method for those functions.

**Comment by jessecambon** [comment]
Yes, reverse geocoding with the Census API isn't currently covered by the package. However, you could use [a script like this one](https://github.com/jessecambon/tidygeocoder/blob/census-reverse-geocoding/sandbox/census_reverse_test.R) to run your query. Here's what the final output for a single input (one set of coordinates) looks like (when consolidated in a single dataframe):

![image](https://user-images.githubusercontent.com/31526206/157354301-f4efb615-a61f-4612-b823-6d8ab6538cd0.png)

The geocodio service can also be used to return Census tract data so that could be worth a look as well.
